staging/src/k8s.io/api/testdata/v1.18.0/events.k8s.io.v1beta1.Event.yaml

```diff
@@ -60,5 +60,4 @@ reportingInstance: "20"
 series:
   count: -1971381490
   lastObservedTime: "2429-02-18T17:57:56.343118Z"
-  state: 鯹)晿<
```

Comment:
actually commit the .after_roundtrip files, don't modify the originals...

Reply:
Done - sorry, didn't realize we should commit those.